src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -72,6 +72,7 @@ export class DictionaryConcepts extends Component {
     localStorage.setItem('type', this.props.match.params.type);
     localStorage.setItem('typeName', this.props.match.params.typeName);
     localStorage.setItem('dictionaryName', this.props.match.params.dictionaryName);
+    localStorage.setItem('params', this.props.match.params);
```

Comment:
@dessHub, I think it would be best if all props were destructured.

Reply:
@abulojoshua1 I only added one line so I didn't want to destructure since line 72 to 74 wasn't mine. I think it is best if we handle it in an independent ticket.